handsontable/src/plugins/copyPaste/clipboardData/copyClipboardData.js

```diff
@@ -0,0 +1,52 @@
+import { META_HEAD, ClipboardData } from './clipboardData';
+import { normalizeRanges } from '../copyableRanges';
+import { getDataByCoords, getHTMLByCoords } from '../../../utils/parseTable';
+
+/**
+ * Creates an object containing information about copy/cut action.
+ *
+ * @private
+ */
+export class CopyClipboardData extends ClipboardData {
+  /**
+   * Sanitized data of "text/html" type inside the clipboard.
+   *
+   * @type {string}
+   */
+  html;
+  /**
+   * Copied data stored as array of arrays.
+   *
+   * @type {string[][]}
+   */
+  data;
+  /**
+   * Copied cell ranges related to instance of Handsontable.
+   *
+   * @type {Array<{startRow: number, startCol: number, endRow: number, endCol: number}>}
+   */
+  copyableRanges;
```

Comment:
```suggestion
  /**
   * Sanitized data of "text/html" type inside the clipboard.
   *
   * @type {string}
   */
  html;
  /**
   * Copied data stored as array of arrays.
   *
   * @type {string[][]}
   */
  data;
  /**
   * Copied cell ranges related to instance of Handsontable.
   *
   * @type {Array<{startRow: number, startCol: number, endRow: number, endCol: number}>}
   */
  copyableRanges;
```
Can these fields be made private? If not, then mark them as private using the JSDoc tag, at least.

Reply:
There is some problem with it, as `html` and `data` elements are inherited. I've marked them as `@private`.